src/VirtualGoogleAssistant.ts

```diff
@@ -92,6 +97,20 @@ export class VirtualGoogleAssistantBuilder {
         return this;
     }
 
+    /**
+     * The name of the handler, or the handler referring to an express server file and the port it's using<br>
+     * The name should just include the file name<br>
+     * `index` is the name of the file - such as index.js<br>
+     * @param {string} handlerName
+     *        {number} port
+     * @returns {VirtualGoogleAssistantBuilder}
+     */
+    public expressHandler(handlerName: string, port: number): VirtualGoogleAssistantBuilder {
```

Comment:
This is just a module name, not a handler name. I would call it `expressModule`. The docs should reflect that we expect the module to do a module.exports on server.